### PR TITLE
GCP cluster

### PR DESCRIPTION
**Pull request by shahargalon**

### CODE REVIEW

**Comment by shahargalon** [comment]
Hi @nirgn975 

I had done all this things and it work, you can try to  GET from book-store.shahargn.com
image

also I put on the deployment file on the image key some string ("IMGAE_VAR"),  that afterwards on the cloudbuild I change it with sed command to the right image (with the updated SHA). it work very nice, I wondering if you also do somthing like that on your cluster..

File: cloudbuild.yml [hunk 1]
```diff
@@ -1,4 +1,20 @@
 steps:
 - name: 'gcr.io/cloud-builders/docker'
-  args: ['build', '-t', 'gcr.io/sodium-dynamo-292520/book-store', '.']
-images: ['gcr.io/sodium-dynamo-292520/book-store']
+  args: ['build', '-t', 'gcr.io/$PROJECT_ID/book-store:$SHORT_SHA', '.']
+- name: 'gcr.io/cloud-builders/docker'
+  args: ['push', 'gcr.io/$PROJECT_ID/book-store:$SHORT_SHA']
+
+- name: ubuntu
```
Comment (nirgn975) [inline cloudbuild.yml:7]: You don't need this step.

File: k8s/book-store-deployment.yaml [hunk 2]
```diff
@@ -17,8 +17,8 @@ spec:
     spec:
       containers:
       - name: book-store
-        image: book-store
-        imagePullPolicy: IfNotPresent
+        image: IMAGE_VAR
```
Comment (nirgn975) [inline k8s/book-store-deployment.yaml:20]: You can just construct the string right here, you have all the environment variables you need (I think, I read it in the docs).
`image: IMAGE_VAR` -> `image: gcr.io/$PROJECT_ID/book-store:$SHORT_SHA`.

**Review by nirgn975 — CHANGES_REQUESTED** [review]

File: cloudbuild.yml (hunk 1 above)
Comment (shahargalon) [reply cloudbuild.yml:7]: I didnt change ot in the end..

**Review by nirgn975 — APPROVED** [review]

**Pull request merged**